apps/openchallenges/challenge-service/src/main/resources/db/challenges.csv

```diff
@@ -462,7 +462,7 @@
-"464","prepare-phase-2-build-it","PREPARE Phase 2 - Build IT!","Help NIH break new ground in early Alzheimer's prediction and related dementias","The goal of the PREPARE Challenge (Pioneering Research for Early Prediction of Alzheimer's and Related Dementias EUREKA Challenge) is to inform novel approaches to early detection that might ultimately lead to more accurate tests, tools, and methodologies for clinical and research purposes. Advances in artificial intelligence (AI), machine learning (ML), and computing ecosystems increase possibilities of intelligent data collection and analysis, including better algorithms and methods that could be leveraged for the prediction of biological, psychological (cognitive), socio-behavioral, functional, and clinical changes related to AD/ADRD.  This second phase, Build IT!: Algorithms and Approaches, is focused on advancing algorithms and analytic approaches for early prediction of AD/ADRD, with an emphasis on explainability of predictions.","","","upcoming","19","","2024-09-01","\N","\N","2023-11-17 00:09:25","2023-12-06 7:18:18"
+"464","prepare-phase-2-build-it","PREPARE Phase 2 - Build IT!","Help NIH break new ground in early Alzheimer's prediction and related dementias","The goal of the PREPARE Challenge (Pioneering Research for Early Prediction of Alzheimer's and Related Dementias EUREKA Challenge) is to inform novel approaches to early detection that might ultimately lead to more accurate tests, tools, and methodologies for clinical and research purposes. Advances in artificial intelligence (AI), machine learning (ML), and computing ecosystems increase possibilities of intelligent data collection and analysis, including better algorithms and methods that could be leveraged for the prediction of biological, psychological (cognitive), socio-behavioral, functional, and clinical changes related to AD/ADRD.  This second phase, Build IT!: Algorithms and Approaches, is focused on advancing algorithms and analytic approaches for early prediction of AD/ADRD, with an emphasis on explainability of predictions.","","","","19","","2024-09-01","\N","\N","2023-11-17 00:09:25","2023-12-06 7:18:18"
```

Comment:
@gaiaandreoletti It looks like this challenge lost its status. Given that the start date is Sept 1st, should it be set to "active"?